pkg/apiserver/certificate/config.go

```diff
@@ -26,20 +30,17 @@ type CAConfig struct {
 	// certificate of antrea-controller.
 	CAConfigMapName string
 
-	// APIServiceNames contains all the APIServices backed by antrea-controller.
-	APIServiceNames []string
-
-	// ValidatingWebhooks contains all the ValidatingWebhookConfigurations backed by antrea-controller.
-	ValidatingWebhooks []string
+	// APIServiceSelector provides the label to select APIService backed by antrea-controller.
```

Comment:
Could you add a comment to explain why we use label but not resource name to select resources?

Reply:
added.